draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -265,7 +265,7 @@ An SD-JWT is composed of
 
 An SD-JWT+KB is composed of
 
-* an SD-JWT, and
+* an SD-JWT (i.e., an Issuer-signed JWT and zero or more Disclosures), and
 * a Key Binding JWT.
```

Comment:
```suggestion
* an Issuer-signed JWT,
* zero or more Disclosures, and
* a Key Binding JWT.
```
This is the most obvious to me.

Reply:
I proposed the above to clarify that the SD-JWT+KB contains an SD-JWT. But I'm fine with either.

Reply:
I think the wording that Daniel is more accurate and reflective of what the intent of the +kb terminology.